app/src/main/kotlin/com/simplemobiletools/dialer/adapters/PhonesAdapter.kt

```diff
@@ -0,0 +1,40 @@
+package com.simplemobiletools.dialer.adapters
+
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.simplemobiletools.commons.models.PhoneNumber
+import com.simplemobiletools.dialer.R
+import com.simplemobiletools.dialer.activities.SimpleActivity
+import kotlinx.android.synthetic.main.item_phone_number.view.select_phone_number_holder
+import kotlinx.android.synthetic.main.item_phone_number.view.select_phone_number_radio
+
+class PhonesAdapter(
```

Comment:
guess this file isnt needed at all anymore, if we use the casual radiogroup dialog